tests/data/transcription/est00.txt

```diff
@@ -0,0 +1,5 @@
+0.120   0.290   225.000
```

Comment:
:) I'm actually pretty rushed, but I also need to write a JAMS wrapper around this module, and a MIDI-to-JAMS converter (counting on ``pretty_midi`` for that one), so I figure it's better to work on those while I wait for the alg output rather than try to produce it myself, which (if past experience is anything to go by) is unlikely to work right off the bat.